clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -119,18 +120,50 @@ public static FetchResponse parse(ByteBuffer buffer, short version) {
         return new FetchResponse(new FetchResponseData(new ByteBufferAccessor(buffer), version));
     }
 
+    private LinkedHashMap<TopicPartition, FetchResponseData.PartitionData> toResponseDataMap(Map<Uuid, String> topicIdToNameMap, short version) {
```

Comment:
We choose to cache responseData here but not in FetchRequest. Is there a particular reason for this inconsistency?

Reply:
I think this inconsistency existed before I touched the code. 😅